pkgs/os-specific/linux/udisks/2-default.nix

```diff
@@ -22,6 +22,10 @@ stdenv.mkDerivation rec {
       substituteInPlace src/main.c --replace \
         "@path@" \
         "${utillinux}/bin:${mdadm}/sbin:/var/run/current-system/sw/bin:/var/run/current-system/sw/bin"
+      substituteInPlace data/80-udisks2.rules \
+        --replace "/bin/sh" "${stdenv.shell}" \
+        --replace "/sbin/mdadm" "${mdadm}/sbin/mdadm" \
+        --replace " sed " " ${gnused}/bin/sed "
```

Comment:
Looking at these expressions:
- I don't understand why `/var/run/current-system/sw/bin` is twice in there (I know this PR doesn't touch that).
- I would personally replace `sbin` occurrences by `bin` (everywhere, eventually), but that's perhaps up to wider discussion. Note: https://github.com/NixOS/nixpkgs/tree/master/pkgs/build-support/setup-hooks/move-sbin.sh